src/newcharacter.cpp

```diff
@@ -2109,6 +2114,14 @@ static std::string assemble_profession_details( const avatar &u, const input_con
             assembled += prof->name() + "\n";
         }
     }
+    // Recipes
+    std::vector<recipe_id> prof_recipe = sorted_profs[cur_id]->recipes();
+    if( !prof_recipe.empty() ) {
+        assembled += "\n" + colorize( _( "Profession recipes:" ), COL_HEADER ) + "\n";
```

Comment:
If you are displaying profession recipes, you should probably display all known recipes (those autolearned from profession skills too) and probably display customized character recipes (based on profession and selected skills).

Reply:
Not sure this is doable at profession selection. Because the character doesn't exist yet, it isn't possible to check if the requirements for a recipe have been met by the player. I am also concerned about displaying a huge number of default recipes for each profession that do not require any skills to know.

Reply:
What's the point of showing specific subset of recipes then?

Reply:
I don't get why we would display this specific recipe and won't display any other recipes during character creation.

Reply:
You can't check for skill requirements for recipes because the player hasn't been assigned any skill increases yet. So if the profession gives a bonus to a skill, you can't check that as far as I know. You can show every recipe that isn't a practice recipe, isn't marked with a SECRET FLAG, or defined in a profession in the professions JSON file (with this PR) but that's it. I would need a way to check for skill requirements without a fully initialized player character.